util/Migrator/DbScripts/2022-09-20_00_AvatarColor.sql

```diff
@@ -0,0 +1,244 @@
+--Add column
+IF COL_LENGTH('[dbo].[User]', 'AvatarColor') IS NULL
+BEGIN
+    ALTER TABLE [dbo].[User] ADD [AvatarColor] VARCHAR (7) NULL;
+END
+GO
+
+-- Recreate VIEW UserView
+IF EXISTS(SELECT * FROM sys.views WHERE [Name] = 'UserView')
+    BEGIN
+        DROP VIEW [dbo].[UserView]
+    END
+GO
+
+CREATE VIEW [dbo].[UserView]
+AS
+SELECT
+    *
+FROM
+    [dbo].[User]
+GO
```

Comment:
Similar to Todd's feedback on [this PR](https://github.com/bitwarden/server/pull/2394#discussion_r1016691383), is it possible to use CREATE OR ALTER VIEW here?